Python/Algorithms/data_structures/linked_list_structs/linked_list.py

```diff
@@ -88,7 +91,7 @@ def push_back(instance, value):
         while temp_node.next is not None:
             temp_node = temp_node.next
 
-        temp_node += instance.make_node(value)
```

Comment:
I have written ```__add__``` for this reason

Reply:
It wasn't working so I changed it. But I have updated __add__ and now it's functional. So I have changed it back to original.